### ISSUE TITLE
Can't access spam queues after updating to 3.0.1 and running commands

### ISSUE THREAD

**Issue by twd3** [silentzco/statamic-akismet#25]
I updated locally then pushed to the server and ran the two commands via SSH. Now when I go to the CP and try and view spam I get:
```
[2021-01-07 18:12:09] production.ERROR: Unable to parse (near "a:6:{s:10:"first_name";s:9:"Salvatore";s:9:"last_name";s:12:"Westmoreland";s:5:"email";s:31:"salvatorewestmoreland@gmail.com";s:5:"phone";s:13:"024 445 21 92";s:9:"q_message";s:14:"Great article.";s:9:"came_from";s:9:"Salvatore";}"). {"userId":"0e51cd0d-d8ef-4960-abbb-b8ceeaee1566","exception":"[object] (Statamic\\Yaml\\ParseException(code: 0): Unable to parse (near \"a:6:{s:10:\"first_name\";s:9:\"Salvatore\";s:9:\"last_name\";s:12:\"Westmoreland\";s:5:\"email\";s:31:\"salvatorewestmoreland@gmail.com\";s:5:\"phone\";s:13:\"024 445 21 92\";s:9:\"q_message\";s:14:\"Great article.\";s:9:\"came_from\";s:9:\"Salvatore\";}\"). at /home/forge/site.com/storage/statamic/tmp/yaml-94bbbcf082af1d4f6b1c2639fb3dffae:1)
[stacktrace]
#0 /home/forge/site.com/vendor/statamic/cms/src/Yaml/Yaml.php(169): Statamic\\Yaml\\ParseException('Unable to parse...', 0, 1, '/home/forge/www...', 1, Object(Exception))
#1 /home/forge/site.com/vendor/statamic/cms/src/Yaml/Yaml.php(171): Statamic\\Yaml\\Yaml->viewException()
#2 /home/forge/site.com/vendor/statamic/cms/src/Yaml/Yaml.php(59): Statamic\\Yaml\\Yaml->validateString()
#3 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(261): Statamic\\Yaml\\Yaml->parse()
#4 /home/forge/site.com/vendor/silentz/akismet/src/Http/Controllers/API/ListSpamController.php(26): Illuminate\\Support\\Facades\\Facade::__callStatic()
#5 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Support/helpers.php(263): Silentz\\Akismet\\Http\\Controllers\\API\\ListSpamController->Silentz\\Akismet\\Http\\Controllers\\API\\{closure}()
#6 /home/forge/site.com/vendor/silentz/akismet/src/Http/Controllers/API/ListSpamController.php(27): tap()
#7 [internal function]: Silentz\\Akismet\\Http\\Controllers\\API\\ListSpamController->Silentz\\Akismet\\Http\\Controllers\\API\\{closure}()
#8 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Collections/Collection.php(630): array_map()
#9 /home/forge/site.com/vendor/silentz/akismet/src/Http/Controllers/API/ListSpamController.php(28): Illuminate\\Support\\Collection->map()
#10 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): Silentz\\Akismet\\Http\\Controllers\\API\\ListSpamController->__invoke()
#11 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Illuminate\\Routing\\Controller->callAction()
#12 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Route.php(254): Illuminate\\Routing\\ControllerDispatcher->dispatch()
#13 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Route.php(197): Illuminate\\Routing\\Route->runController()
#14 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Router.php(692): Illuminate\\Routing\\Route->run()
#15 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Routing\\Router->Illuminate\\Routing\\{closure}()
#16 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CP/CountUsers.php(18): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#17 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CP\\CountUsers->handle()
#18 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CP/Localize.php(16): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#19 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CP\\Localize->handle()
#20 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CP/Authorize.php(25): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#21 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CP\\Authorize->handle()
#22 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CP/ContactOutpost.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#23 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CP\\ContactOutpost->handle()
#24 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CP/ControlPanel.php(16): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#25 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CP\\ControlPanel->handle()
#26 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(41): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#27 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Routing\\Middleware\\SubstituteBindings->handle()
#28 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(78): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#29 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\VerifyCsrfToken->handle()
#30 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#31 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\View\\Middleware\\ShareErrorsFromSession->handle()
#32 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#33 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(63): Illuminate\\Session\\Middleware\\StartSession->handleStatefulRequest()
#34 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Session\\Middleware\\StartSession->handle()
#35 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#36 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\AddQueuedCookiesToResponse->handle()
#37 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#38 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Cookie\\Middleware\\EncryptCookies->handle()
#39 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/SwapExceptionHandler.php(19): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#40 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\SwapExceptionHandler->handle()
#41 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#42 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Router.php(694): Illuminate\\Pipeline\\Pipeline->then()
#43 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Router.php(669): Illuminate\\Routing\\Router->runRouteWithinStack()
#44 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\\Routing\\Router->runRoute()
#45 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Routing/Router.php(624): Illuminate\\Routing\\Router->dispatchToRoute()
#46 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(166): Illuminate\\Routing\\Router->dispatch()
#47 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(128): Illuminate\\Foundation\\Http\\Kernel->Illuminate\\Foundation\\Http\\{closure}()
#48 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/CheckMultisite.php(14): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#49 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\CheckMultisite->handle()
#50 /home/forge/site.com/vendor/statamic/cms/src/Http/Middleware/PoweredByHeader.php(19): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#51 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Statamic\\Http\\Middleware\\PoweredByHeader->handle()
#52 /home/forge/site.com/vendor/barryvdh/laravel-debugbar/src/Middleware/InjectDebugbar.php(60): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#53 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Barryvdh\\Debugbar\\Middleware\\InjectDebugbar->handle()
#54 /home/forge/site.com/vendor/fideloper/proxy/src/TrustProxies.php(57): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#55 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Fideloper\\Proxy\\TrustProxies->handle()
#56 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#57 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#58 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/TransformsRequest.php(21): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#59 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\TransformsRequest->handle()
#60 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/ValidatePostSize.php(27): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#61 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\ValidatePostSize->handle()
#62 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#63 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(167): Illuminate\\Foundation\\Http\\Middleware\\PreventRequestsDuringMaintenance->handle()
#64 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): Illuminate\\Pipeline\\Pipeline->Illuminate\\Pipeline\\{closure}()
#65 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(141): Illuminate\\Pipeline\\Pipeline->then()
#66 /home/forge/site.com/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(110): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter()
#67 /home/forge/site.com/public/index.php(55): Illuminate\\Foundation\\Http\\Kernel->handle()
#68 {main}
```

**Comment by twd3** [comment]
Side note: I tried to SSH in and go to folder /storage/statamic/tmp to delete spam manually but the only thing in there is another folder called forms and it's empty.

**Comment by edalzell** [comment]
Can you please send me your existing spam that is in the `storage/app/spam` folder? Please zip it up and send.

If you'd like to email, erin@silentz.co

Thanks

**Comment by twd3** [comment]
Okay, sending it over now. I was able to delete everything in there after downloading. Hopefully, that fixes this issue since I think it was just a conversion thing.

**Comment by twd3** [comment]
I just tried updating to 3.0.38 and couldn't...got this error:

```
PHP Fatal error:  Access level to Silentz\Akismet\ServiceProvider::bootActions() must be protected (as in class Statamic\Providers\AddonServiceProvider) or weaker in C:\_statamic\v-3\vendor\silentz\akismet\src\ServiceProvider.php on line 58In ServiceProvider.php line 58:

  Access level to Silentz\Akismet\ServiceProvider::bootActions() must be protected (as in class Statamic\Providers\Ad
  donServiceProvider) or weaker


Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 255
```

**Comment by edalzell** [comment]
fixed in 3.0.2
